mmv1/products/networkservices/TcpRoute.yaml

```
@@ -181,3 +181,9 @@ properties:
               name: originalDestination
               description: |
                 If true, Router will use the destination IP and port of the original connection as the destination of the request.
+            - !ruby/object:Api::Type::String
+              name: idleTimeout
+              description: |
+                Specifies the idle timeout for the selected route. The idle timeout is defined as the period in which there are no bytes sent or received on either the upstream or downstream connection.
```

I figured it would be worth including the full field description so that users know about the validation when reading the provider docs, and know that a default value is provided by the API when unspecified.

```suggestion
                Specifies the idle timeout for the selected route. The idle timeout is defined as the period in which there are no bytes sent or received on either the upstream or downstream connection. If not set, the default idle timeout is 30 seconds. If set to 0s, the timeout will be disabled.

                A duration in seconds with up to nine fractional digits, ending with 's'. Example: "3.5s".
```